### ISSUE TITLE
Add basemap selection button to map

### ISSUE THREAD

**Issue by amyleew** [amyleew/osrm-frontend#5]
with [Leaflet Layer.control](http://leafletjs.com/examples/layers-control.html)

![image](https://cloud.githubusercontent.com/assets/4587826/8583944/a50dc996-259f-11e5-8a09-b4119f1ed664.png)

**Comment by amyleew** [comment]
https://gist.github.com/tristen/1914fd00b59523aac1e0

**Comment by amyleew** [comment]
@tristen - Basemaps are showing up and swapping out but have questions:

![basemaps](https://cloud.githubusercontent.com/assets/4587826/8601069/a0873220-2637-11e5-8b23-e0bf49a1750e.gif)
- [x]  Seems like each map is just show / hiding layers instead of replacing based on selection. Can I control loading so it's done individually? 
- [x] `layer.options.id` holds the proper map label how do I set that to the control to replace the numbers?
- [ ] How do I add `position` from `L.control.layers(mapLayer).addTo(map);` Not sure on the syntax, I tried:

``` L.control({
  .layers(mapLayer),
  .setPosition('topright')
}).addTo(map);
```
